docs/sample_config.yaml

```diff
@@ -1084,13 +1084,16 @@ signing_key_path: "CONFDIR/SERVERNAME.signing.key"
 #    verify_keys:
 #      "ed25519:auto": "abcdefghijklmnopqrstuvwxyzabcdefghijklmopqr"
 #  - server_name: "my_other_trusted_server.example.com"
+
+trusted_key_servers:
+- server_name: "matrix.org"
+
+# 'trusted_key_servers' defaults to matrix.org, but using it will generate a
```

Comment:
this should be in the documentation for `trusted_key_servers`, not `suppress_key_server_warning`